tools/scripts/rebuildAnimationLibraryManifest.rb

```diff
@@ -116,6 +116,8 @@ def get_animation_objects(bucket)
     bucket.objects.each do |object_summary|
       animation_name = object_summary.key[/^[^.]+/]
       extension = object_summary.key[/(?<=\.)\w+$/]
+      next if extension.nil? # Skip 'directory' objects
```

Comment:
This skips directories, which are normally included when you ask S3 to 'list all objects in bucket'.